README.md

```diff
@@ -28,6 +28,7 @@ index: {
   //The name of the routes if you are using the lazy-route mixin, no minisearch expressions are allowed here.
   routes: ["index", "..."]
   //The dependencies for this bundle. They will loaded in the same batch as the actual bundle
+  // The dependencies had to be loaded prior to the dependent bundles.
```

Comment:
In dependencies are added after the dependent bundle in `config/bundles.js`, only the dependencies bundles will be loaded for routes.